sdk/ml/azure-ai-ml/azure/ai/ml/entities/_workspace/workspace.py

```diff
@@ -251,6 +252,7 @@ def _from_rest_object(cls, rest_obj: RestWorkspace) -> "Workspace":
             feature_store_settings=feature_store_settings,
             enable_data_isolation=rest_obj.enable_data_isolation,
             workspace_hub=rest_obj.hub_resource_id,
+            workspace_id=rest_obj.workspace_id,
```

Comment:
If we are expecting rest_obj to have a "workspace_id" in the from_rest_object method should workspace_id be passed in the to_rest_object method? Looks to me like workspace_id will just be set to None here.

Reply:
Good question: I think this ID is set by the service during the creation, so we don't explicitly pass it.